cluster/src/main/java/org/apache/iotdb/cluster/log/manage/RaftLogManager.java

```diff
@@ -125,6 +127,8 @@ protected RaftLogManager(StableEntryManager stableEntryManager, LogApplier appli
     long first = getCommittedEntryManager().getDummyIndex();
     long last = getCommittedEntryManager().getLastIndex();
     this.setUnCommittedEntryManager(new UnCommittedEntryManager(last + 1));
+    this.getUnCommittedEntryManager()
+        .truncateAndAppend(stableEntryManager.getAllEntriesAfterCommittedIndex());
```

Comment:
Actually, these entries are committed at a high level. So we need to `commit` it before we expose them to the outside world, which means we need to revocer them in `commitEntryManager`, and `unCommitEntryManager` should be empty at current design.

Reply:
> which means we need to revocer them in `commitEntryManager`,

That means,we need to update meta's info when recover logs file in this case(meta's info is stale)? that may be difficult _(because we need to consider the meta recovery process in all situations to ensure that leaders and followers are completely consistent)_.

Of course,there is another way to fix it. Every time the log file is flushed, the meta is flushed with it. Except in special 
 situations _(log file flushed successfully but meta failed when we restart cluster)_, it doesn't have any problems.

How do you think?

Reply:
If we update the log and meta information of all nodes to the status of all logs persisted and applied after restart, will there be any inconsistency between the leader and follower?

Reply:
This could not cause consistency problems, but some data may be lost. 
For example, `applyIndex` in persistent meta file is 5, and `commitIndex` is 7.But the actual index flushed to 10,and apply to 7,we may lost 8~10.

Reply:
OK, I see. This is really a matter of ensuring atomicity between apply and commit. We can talk about it later during refactoring, I have no problem with the current implementation

Reply:
In my opinion, it doesn't matter where the entries to be recovered. The recover progress won't affect the consistency of raft group. `CommitIndex` to be determined until the next new log appending the follower. If my understanding right, the `committed`  logs committed again in the same sequence won't break the correctness of IoTDB. We have no choice even though this is not a good one.